config/config.yaml

```diff
@@ -4,6 +4,15 @@ kraken:
         -    scenarios/etcd.yml
         -    scenarios/openshift-kube-apiserver.yml                           
         -    scenarios/openshift-apiserver.yml
+    node_scenarios:
+        node_start_stop_scenario:
+            enable_scenario:                               # Set to True to enable the scenario
+            selector:                                      # This enables the nodes with the specified selector to be selected for chaos injection
+            number_of_kills:                               # Number of times to inject the scenario
+        node_termination_scenario:
```

Comment:
Maybe combine the start_stop scenario and terminate into one since they both need the same details ( selector, number_of_kills )? Something like:
node_scenarios:
    action: start/stop/terminate
    selector:
    number_of_kills:

Thoughts?